modular_citadel/code/modules/clothing/suits/suits.dm

```diff
@@ -40,4 +40,17 @@
 	body_parts_covered = CHEST
 	resistance_flags = NONE
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
-	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -5, "acid" = -15) //nylon sucks against acid
\ No newline at end of file
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -5, "acid" = -15) //nylon sucks against acid
+
+/obj/item/clothing/suit/assu_suit
+	name = "ASSU suit"
+	desc = "A cheap replica of old SWAT armor. On its back, it is written: \"<i>Assistance Skipping Special Unit</i>\"."
+	icon = 'modular_citadel/icons/obj/clothing/assu.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/mob/clothing/assu.dmi'
```

Comment:
Same, move this item to `code/modules/clothing/suits/miscellaneous.dm`, remove these two lines and add the sprites to `clothing/suits.dmi` and `mob/suit.dmi` instead.